core/src/smtp/mod.rs

```diff
@@ -331,10 +333,12 @@ async fn smtp_is_catch_all(
 	domain: &str,
 ) -> Result<bool, SmtpError> {
 	// Create a random 15-char alphanumerical string.
-	let random_email = rand::thread_rng()
-		.sample_iter(&Alphanumeric)
+	let mut rng = SmallRng::from_entropy();
+	let random_email: String = iter::repeat(())
+		.map(|()| rng.sample(Alphanumeric))
+		.map(char::from)
 		.take(15)
-		.collect::<String>();
```

Comment:
In case you're wondering why I changed that: `.sample_iter(&Alphanumeric)` returns `u8` now, so we have to convert that to a `char` first to collect it into a String (because u8 doesn't impl Into<String>).
Also the recommended syntax slightly changed according to the [Rust cookbook](https://rust-lang-nursery.github.io/rust-cookbook/algorithms/randomness.html#create-random-passwords-from-a-set-of-alphanumeric-characters).